esx_service/utils/kvESX.py

```diff
@@ -256,32 +257,42 @@ def load(volpath):
     """
     Load and return dictionary from the sidecar
     """
+    logging.info("Load dictionary from sidecar for vol path %s", volpath)
```

Comment:
so this is for a bug hunting ? Then there should be an issue to roll it back when the bug is found. 

I do not think we should print that much information which really has no value outside of catching this bug.